gdcdictionary/schemas/exposure.yaml

```diff
@@ -283,6 +318,8 @@ properties:
       - Smokeless Tobacco
       - Tobacco
       - Wood Dust
+      - Coal Dust
+      - Diesel Exhaust
```

Comment:
alphabeticize

Reply:
oof, this just got a bit more complicated.  I see that we have "chemical_exposure_type", which contains "Diesel Exhaust".  Should this be removed from "exposure_type"?  Also should we add "Coal Dust" as a "chemical_exposure_type" and remove it from "exposure_type"?

@tlicht3

Reply:
I don't think we meant to add "Diesel Exhaust" to exposure_type because, as you noted, that's redundant... must have been a typo or oversight of some kind - it should only be in the chemical_exposure_type. i suppose we could also add "Coal Dust" there, but it seems more like a toxic substance, which is what a lot of the other values are in the exposure_type (perhaps in a future release we could work on that?). I think the easiest thing to do here is just remove "Diesel Exhaust" and alphabetize "Coal Dust", but happy to discuss. further.

Reply:
@tlicht3  @wwysoc2 sounds great, I'll remove "Diesel Exhaust" and alphabetize "Coal Dust"